libs/frontend/feature-build/src/lib/style/use-reusable-style-menu.ts

```diff
@@ -133,6 +134,7 @@ export const useReusableStyleMenu = (): IUseStyleMenuFeature => {
       breakpointStylesWithSource,
       descSortedBreakpoints.value,
       activeBreakpoint.value,
+      currentStyleType.value === StyleType.Mixin,
```

Comment:
For #205 I just added an optional arg to decide if `sourcePseudoClass` should be included in the returned value of `computeComponentFlattenedStyles`. I hope this is okay 😬.